src/components/UiCardItem/styles.ts

```diff
@@ -1,4 +1,4 @@
-export const cardItemWhyUsStyles = {
+export const largeCardItemStyles = {
```

Comment:
create a folder for small card and create a file for styles and after move this styles there